src/components/footer/styledFooter.js

```diff
@@ -1,20 +1,50 @@
 import styled from "styled-components";
 
-export const FooterWrapper = styled.footer`
-    display: flex;
-    flex-direction: column;
+export const FooterSection = styled.footer`
     background: #1C383C;
     padding: 80px 0 60px 0;
+
+    @media(max-width: 600px) {
+        padding-bottom: 40px;
+    }
+`
+export const FooterWrapper = styled.div`
+    width: 88.89%;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+
+    @media(max-width: 1024px) {
+        width: 84.38%;
+    }
 `
 export const FooterBlocks = styled.div` 
     display: flex;
-    width: 88.89%;
     justify-content: space-between;
+    width: 100%;
     margin: auto;
+
+    @media (max-width: 1100px) {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(2, minmax(238px, 1fr));
+        grid-column-gap: 180px;
+    }
+
+    @media(max-width: 768px) {
+        display: flex;
+        flex-direction: column;
+    }
 `
 export const Block = styled.div`
     display: flex;
     flex-direction: column;
+    order: ${props => props.$order && 3};
+
+    @media(max-width: 768px) {
+        border-bottom: 1px solid white;
```

Comment:
I think we should remove this since it isn't in the design. We could probably get away with this for the 768 size since we have nothing to go off, but the design doesn't have a border-bottom at 375.